tool/templates/kotlin/init.kt.jinja

```diff
@@ -39,6 +41,26 @@ object DW {
     }
 }
 
+internal interface DiplomatJVMRuntimeLib: Library {
+    fun create_rust_jvm_cookie(env: JNIEnv, obj: Object): Pointer
+    fun destroy_rust_jvm_cookie(obj_pointer: Pointer): Unit
+}
+
+internal class DiplomatJVMRuntime {
+    companion object {
+        val libClass: Class<DiplomatJVMRuntimeLib> = DiplomatJVMRuntimeLib::class.java
+        val lib: DiplomatJVMRuntimeLib = Native.load("{{lib_name}}", libClass, Collections.singletonMap(Library.OPTION_ALLOW_OBJECTS, true))
+
+        fun getRustCookie(obj: Object): Pointer {
```

Comment:
Nit: `get` seems odd because if you all it twice on the same object, you get two cookies, not the same one. Maybe `build`?